bitnami/grafana-operator/templates/servicemonitor.yaml

```
@@ -24,28 +21,23 @@ spec:
       targetPort: metrics
       protocol: TCP
       name: metrics
-  selector: {{- include "common.labels.matchLabels" . | nindent 4 }}
+  {{- $podLabels := merge .Values.operator.podLabels .Values.commonLabels }}
+  selector: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 4 }}
 ---
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ include "common.names.fullname" . }}
   namespace: {{ default (include "common.names.namespace" .) .Values.operator.prometheus.serviceMonitor.namespace }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-    {{- if .Values.operator.prometheus.serviceMonitor.labels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.operator.prometheus.serviceMonitor.labels "context" $ ) | nindent 4 }}
-    {{- end }}
-    {{- if .Values.commonLabels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
-    {{- end }}
-  annotations:
-    {{- if .Values.commonAnnotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-    {{- end }}
+  {{- $labels := merge .Values.operator.prometheus.serviceMonitor.labels .Values.commonAnnotations }}
```

I guess you mean `commonLabels`

```suggestion
  {{- $labels := merge .Values.operator.prometheus.serviceMonitor.labels .Values.commonLabels }}
```

---

Done at https://github.com/bitnami/charts/pull/18305/commits/3ef29bd86d13dce31f039e13b48b1b712a163f2b